source/scss/style.scss

```diff
@@ -86,6 +86,3 @@
 
 // Form Checkbox
 @import "components/form-checkbox";
-
```

Comment:
the checkbox was also added for QS. But if Planets want to keep it, ok...

Reply:
Happened with so many other elements. Checkboxes are a very general input element. Can be left here.